tests/test_bigquery_eager.py

```diff
@@ -251,6 +320,29 @@ def tearDownClass(cls):  # pylint: disable=invalid-name
         """setUpClass"""
         cls.server.stop()
 
+    def _get_read_session(self, client):
+        return client.read_session(
+            self.PARENT,
+            self.GCP_PROJECT_ID,
+            self.TABLE_ID,
+            self.DATASET_ID,
+            selected_fields={
+                "string": {"output_type": dtypes.string},
+                "boolean": {"output_type": dtypes.bool},
+                "int": {"output_type": dtypes.int32},
+                "long": {"output_type": dtypes.int64},
+                "float": {"output_type": dtypes.float32},
+                "double": {"output_type": dtypes.float64},
+                "repeated_bool": {"mode": "repeated", "output_type": dtypes.bool},
```

Comment:
leave as a todo note for myself: this test current doesn't test backwards compatibility (but I did manually rollback tests/test_bigquery_eager.py to previous version and test that selected_fields as list + output_types still pass, so the change is backwards compatible).  Let me add a test later.

cc @vlasenkoalexey